RecoBTag/DeepFlavour/python/DeepFlavourExporter_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+
+input_names = ["input_{}:0".format(i) for i in range(1,6)]
+output_names = ["ID_pred/Softmax:0", "regression_pred/BiasAdd:0"]
```

Comment:
how are these relevant to this file to define pfDeepFlavourExporter?

Reply:
not relevant, I will remove the Exporter plugin becuase it is not needed anymore

Reply:
Exporter removed with https://github.com/cms-sw/cmssw/pull/19893/commits/de7e130c318f4dc84a506c2e2aef7d32900013a8